packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/sensororientation/DeviceOrientationManager.java

```diff
@@ -260,6 +265,8 @@ PlatformChannel.DeviceOrientation getUIOrientation() {
         } else {
           return PlatformChannel.DeviceOrientation.LANDSCAPE_RIGHT;
         }
+      case Configuration.ORIENTATION_SQUARE:
```

Comment:
Do we need this if it is deprecated?

Reply:
It's deprecated, but there was a warning for missing a case:

```
/Users/bmparr/Development/packages/packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/features/sensororientation/DeviceOrientationManager.java:255: Error: Switch statement on an int with known associated constant missing case Configuration.ORIENTATION_SQUARE [SwitchIntDef]
    switch (orientation) {
    ~~~~~~

```

The alternative was to suppress the warning that not all of the cases were handled. I thought this would be worse since it wouldn't notify us when a new one was added.